appOPHD/States/GameState.cpp

```diff
@@ -20,15 +20,17 @@ NAS2D::Point<int> MOUSE_COORDS; /**< Mouse Coordinates. Used by other states/wra
 
 GameState::GameState(const std::string& savedGameFilename) :
 	mMainReportsState{std::make_unique<MainReportsUiState>()},
-	mMapViewState{std::make_unique<MapViewState>(*this, savedGameFilename)}
+	mMapViewState{std::make_unique<MapViewState>(*this, savedGameFilename)},
+	mFileIoDialog{{this, &GameState::onLoadGame}}
 {
 	initializeGameState();
 }
 
 
 GameState::GameState(const Planet::Attributes& planetAttributes, Difficulty selectedDifficulty) :
 	mMainReportsState{std::make_unique<MainReportsUiState>()},
-	mMapViewState{std::make_unique<MapViewState>(*this, planetAttributes, selectedDifficulty)}
+	mMapViewState{std::make_unique<MapViewState>(*this, planetAttributes, selectedDifficulty)},
+	mFileIoDialog{{this, &GameState::onLoadGame}}
```

Comment:
Same for this class. When connecting with `Delegate` we should remove the `connect` call for `Signal`.

Edit: Which is in the following commit.